com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/topology.py

```diff
@@ -129,6 +130,21 @@ def _map(self, func, schema):
         oport = op.addOutputPort(schema=schema)
         return Stream(self.topology, oport)
 
+    def view(self, buffer_time = 10.0, sample_size = 10000):
+        """
+        Defines a view on a stream. Returns a view object which can be graphed after the jobs is submitted.
+        """
+        name = ''.join(random.choice('0123456789abcdef') for x in range(16))
```

Comment:
The operator name isn't unique. There can be multiple views on a single port on a single operator. How about a naming scheme like <operator name><view number> ?